drivers/RpiLanPropertyChange/bcm2836/RpiLanPropertyChange.vcxproj

```diff
@@ -81,6 +83,13 @@
       <AdditionalDependencies>onecore.lib;kernel32.lib;user32.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <SetChecksum>true</SetChecksum>
     </Link>
+    <PostBuildEvent>
+      <Command>set SIGN_OEM=1&amp;&amp;set&amp;&amp;"$(WDKContentRoot)\Tools\bin\i386\pkggen.exe" $(TargetName).pkg.xml /config:"$(WDKContentRoot)\Tools\bin\i386\pkggen.cfg.xml" /output:"$(OutDir)\$(TargetName) CBS\\" /version:1.0.0.0 /build:fre /cpu:ARM /variables:"SIGN_OEM=1;OEMNAME=Contoso;BSPARCH=ARM;_RELEASEDIR=$(OutDir);"</Command>
```

Comment:
@parameshbabu Is there a better way to invoke pkggen to generate a CAB for a service? PackageGen does not appear to be an option in the normal VS Studio properties like it is for drivers.

We are trying to include this service as part of the baseline BSP to fix a network MAC address issue.

Reply:
Yes, this is not good way. VS cannot generate packages for exe/dll like drivers. But if use pkggen, environment settings like test certs and signtools are required to setup.